packages/react/src/themes/teams/components/Menu/menuItemStyles.ts

```diff
@@ -402,21 +403,47 @@ const menuItemStyles: ComponentSlotStylesInput<MenuItemPropsAndState, MenuVariab
     }
   },
 
+  content: ({ props: p }): ICSSInJSStyle => {
+    const widthAdjust = (p.icon ? 26 : 0) + (p.menu ? 16 : 0)
+    return {
+      whiteSpace: 'normal',
```

Comment:
I think the default value is already `normal`.. is this `white-space` property overriding something?

Reply:
The root element has `whiteSpace: 'nowrap'` and that seems to affect all children, but the text within the "content" should wrap if necessary, so this returns it to `'normal'`.